stratum/hal/lib/common/hal.cc

```diff
@@ -198,15 +198,14 @@ ::util::Status Hal::Run() {
     for (const auto& url : external_stratum_urls) {
       builder.AddListeningPort(url, server_credentials);
     }
+    constexpr uint32 MB = 1024 * 1024;
```

Comment:
This change kinda indicates that the flag definition is bad.

I think it should be `grpc_max_recv_msg_size` and in bytes or `grpc_max_recv_msg_size_mb`.